api/report.go

```diff
@@ -80,5 +80,11 @@ func (a *api) AddReport(ctx echo.Context, commitSha string, params AddReportPara
 			fmt.Sprintf("couldn't store test results: %s", err))
 	}
 
+	err = database.UpdateFlakyTests(a.db, *commit)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest,
```

Comment:
Shouldn't this be an internal server error?

Reply:
True!